extensions/amp-slikeplayer/OWNERS

```diff
@@ -0,0 +1,14 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
```

Comment:
Someone in Slike player should be added to OWNERS as well.

Reply:
Added detail in OWNERS